scripts/qa_crowbarsetup.sh

```diff
@@ -3647,21 +3645,17 @@ function oncontroller_magnum_service_setup
             --property os_distro=fedora-atomic $service_image_name
     fi
 
+    local service_sles_image_name=openstack-magnum-k8s-image
```

Comment:
Do we need this check and create image again here? I guess mkcloud would anyways fail if something goes wrong when we are doing this in the post_install recipe..